packages/app/src/app/components/SubscribeForm/CheckoutForm/index.tsx

```diff
@@ -4,10 +4,12 @@ import {
   CardElement,
```

Comment:
Same here! Important to check `/patron` and `/pro`.

Reply:
Fixed and working